prov/verbs/src/ep_rdm/verbs_av_ep_rdm.c

```diff
@@ -577,9 +580,12 @@ fi_ibv_rdm_av_map_addr_to_conn_add_new_conn(struct fi_ibv_rdm_ep *ep,
 			HASH_ADD(hh, av_entry->conn_hash, ep,
 				 sizeof(struct fi_ibv_rdm_ep *), conn);
 		}
+err:
```

Comment:
This is misnamed. It's almost never going to be an error.

Reply:
`done` label should be ok